cmd/pmm-managed-starlark/main.go

```diff
@@ -17,13 +17,25 @@
 package main
 
 import (
+	"encoding/json"
 	"log"
 	"os"
 	"strconv"
 
+	"github.com/percona-platform/saas/pkg/starlark"
+	"github.com/percona/pmm/api/agentpb"
 	"github.com/percona/pmm/version"
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
 	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/percona/pmm-managed/services/checks"
+)
+
+const (
+	cpuLimit    = 4                 // 4 seconds of CPU time
+	memoryLimit = 100 * 1024 * 1024 // 100MB of memory in bytes
```

Comment:
That's one of the cases where I think better type and better variable name would be better than a comment: use time.Duration for cpuLimit and `memoryLimitBytes`

Reply:
`unix.Rlimit` expects the limit to be in `uint64` so I change `cpuLimit` to  `time.Duration` will have to convert it back to `uint64` to be able to use it, so I would prefer to let it stay the way it is.

:+1: for `memoryLimitBytes`

Reply:
`uint64(cpuLimit.Seconds())`

Reply:
The comment above has 👍, but is not addressed

Reply:
sorry forgot that, made the change will push